docs/development.md

```diff
@@ -281,15 +282,55 @@ I0511 10:15:29.593106 244858 hello_world.cc:57] Response: HELLO AGAIN WORLDS!
 
 ### Signing Oak Wasm modules
 
-Oak Wasm modules can be signed using [Ed25519](https://ed25519.cr.yp.to/)
-scheme.
+As described in [Oak Concepts](/docs/concepts.md#labels) Oak Wasm modules can be
+signed using [Ed25519](https://ed25519.cr.yp.to/) scheme.
 
-In order create a signature `hello_world.sign` for a Wasm module
-`hello_world.wasm` use the following script:
+If you are reviewing an Oak module (for example `private_set_intersection.wasm`)
+and want to create a signature for it, first you need a private/public key pair.
+It can be generated using `oak_sign` utility:
+
+```bash
+./scripts/oak_sign generate \
+  --private-key=examples/keys/ed25519/test.key \
+  --public-key=examples/keys/ed25519/test.pub
+```
+
+It generates two key files (`test.key` and `test.pub`) that are encoded using
+[PEM](https://tools.ietf.org/html/rfc1421) format.
```

Comment:
I think the fact that they are PEM encoded is irrelevant and an implementation detail. Or am I missing something for which this is useful at all?

Reply:
It's just a clarification, so the reader will understand what are key files. So we would not copy the `minisign` mistake, where documentation doesn't say what format is used for keys.

Reply:
I think in our case, we actually don't want others to use these keys for anything else. I am not sure why we would want to document them at all, if we do then obviously this becomes a public API and we cannot change it any more, which does not seem worthwhile to me. What use cases are you thinking in which someone would parse such keys and use them without relying on oak utilities?

Reply:
Deleted